test/fixtures/rules/std_rule_cases/L057.yml

```diff
@@ -320,3 +320,32 @@ test_pass_alteration_identifier_in_file_reference_spark3:
   configs:
     core:
       dialect: spark3
+
+test_pass_ignore_lists_quoted:
```

Comment:
Re: My earlier comment, can we add tests for upper and/or mixed-case strings?

Reply:
Fixed